rossum_api/models/annotation.py

```diff
@@ -10,12 +10,123 @@
 
 @dataclass
 class Prediction:
+    """Internal only description of prediction source and version."""
+
     source: str
     version: str | None = None
 
 
 @dataclass
 class Annotation:
+    """Annotation contains all extracted and verified data related to a document.
+
+    Every document belongs to a :class:`~rossum_api.models.queue.Queue` and is related
+    to the :class:`~rossum_api.models.schema.Schema`, that defines datapoint types and overall shape
+    of the extracted data.
+
+    Arguments
+    ---------
+    url
```

Comment:
Ouch, Having to maintain the list of attributes up-to-date in so many places is horrible. How about using sphinx-autodoc-typehints plugin to generate it? We would loose the attribute descriptions but it's mostly self-explanatory and we can always add a link to Elis API docs for details.